examples/parameters/src/getting_started_set_ssm_parameter_overwrite.py

```diff
@@ -0,0 +1,17 @@
+from aws_lambda_powertools.utilities import parameters
+from aws_lambda_powertools.utilities.typing import LambdaContext
+
+
+def lambda_handler(event: dict, context: LambdaContext) -> dict:
+    try:
+        # Set a single parameter, but overwrite if it already exists
+        # by default, overwrite is False, explicitly set it to True
```

Comment:
Took me a while to understand the sentences without punctuation :D 

```suggestion
        # Set a single parameter, but overwrite if it already exists.
        # Overwrite is False by default, so we explicitly set it to True
```